docs/src/misc.rst

```diff
@@ -125,6 +125,36 @@ Data types
             } netmask;
         } uv_interface_address_t;
 
+.. c:type:: uv_network_interface_t
+
+    Data type for network interfaces.
+
+    ::
+
+        typedef struct uv_network_interface_s {
+            char* name;
+            int is_up_and_running;
+            int is_loopback;
+            int is_point_to_point;
+            int is_promiscuous;
+            int has_broadcast;
+            int has_multicast;
```

Comment:
Using booleans is painting ourselves into a corner from an API/ABI perspective.  It's arguably better to have a flags field and `#define`s for individual flags/masks.

Alternatively, one or more opaque flag fields with a getter-like function or functions operating on a `uv_network_interface_t*` to extract values.  Somewhat un-libuv-y but probably easier for people using libuv through a foreign function interface.